task/README.adoc

```diff
@@ -0,0 +1,5 @@
+= Spring Cloud Task
+
+We're wrapping `TaskExecutionListener`s and `CommandLineRunner` and `ApplicationRunner`.
```

Comment:
Added a TaskExecutionListener to the sample to see if I could see the output in Zipkin.   I didn't see it appear.   But I probably messed something up.   Here is my additions: https://github.com/cppwfs/spring-cloud-sleuth-samples/commit/465a2ea07a50a7bd8257f2b81975d6b1702eb50c

Reply:
I'll add the info to the `README`